src/PoolManager.sol

```diff
@@ -280,11 +281,6 @@ contract PoolManager is IPoolManager, ProtocolFees, NoDelegateCall, ERC6909Claim
         return pools[id].getPoolBitmapInfo(word);
     }
 
-    /// @notice Temporary view function. Replaceable by transient EXTSLOAD.
-    function getReserves(Currency currency) external view returns (uint256 balance) {
```

Comment:
you should also be able to remove the following functions: `getNonzeroDeltaCount`, `currencyDelta`, `isUnlocked`